src/library/module.cpp

```diff
@@ -568,6 +568,15 @@ environment add_inductive(environment                       env,
     new_env = add_decl_pos_info(new_env, decl.m_name);
     return add(new_env, std::make_shared<inductive_modification>(cidecl, env.trust_lvl()));
 }
+
+optional<declaration> is_decl_modification(modification const & mod) {
```

Comment:
Nitpick, but maybe this should be `as_decl_modification` since the return value is not just a Boolean?

Reply:
I think `is_*` is the common name for such a function in Lean, e.g. [`is_bit0`, etc.](https://github.com/leanprover-community/lean/blob/ec1613aef1eee72e601f192b16740629c6d49690/src/library/num.h#L21):
```c++
optional<expr> is_bit0(expr const & e);
optional<expr> is_bit1(expr const & e);
optional<expr> is_neg(expr const & e);
```